src/utils/renderer.js

```diff
@@ -14,6 +14,26 @@ export const updateLottoAmounts = (selector, amounts) => {
     `;
 };
 
+export const updateLottoTickets = (amounts, lottoTicketsList) => {
+  const newLottoTickets = new Array(amounts).fill(undefined).map(
```

Comment:
spread를 활용해서 fill 없이 써줘도 좋을 것 같습니다.
```suggestion
  const newLottoTickets = [... new Array(amounts)].map(
```
혹은 [...new Array(amounts).keys()] 로 써줘서 (_, idx) 없이 써줘도 좋을 것 같아요

Reply:
그냥 new Array().map을 하면 안되길래 fill로 채웠는데 spread 연산으로 해결할 수 있는 거였네요,, !!